Resources/android/ExampleView.js

```diff
@@ -0,0 +1,20 @@
+var View = require('android.view.View'),
+  Activity = require('android.app.Activity'),
+  TiActivity = new Activity(Ti.Android.currentActivity);
+
+exports.create = function(params) {
+  params = params || {};
+
+  var view = new View(TiActivity);
+
+  if (params.backgroundColor) {
+
+    // Hyperloop should provide cross-platform helpers you can use anywhere in
+    // the app to do stuff like this:
+    var gColor = Hyperloop.colorFromString(params.backgroundColor);
+
+    view.setBackgroundColor(gColor);
+  }
```

Comment:
We should think of a more generic approach on Views. Native modules currently subclass the `TiUIView` which already handles all basic properties (backgroundColor, width, height, ...). But as we want to get rid of the proxy system, we would need to provide some classes that do that beforehand. @pec1985 thoughts?